DTS-STN/canadian-dental-care-plan
# Defaulting environment variables for Canada/USA lookup IDs and hCaptcha `/siteverify` url

### Description
This will reduce the amount of configuration required as these values are unlikely to change per environment.

### Checklist

### Test Instructions
Run the application locally.
hCaptcha call should still happen when continuing from Terms and Conditions page
Selecting "Canada"/"United States" on the Personal Information page should still render the correct provinces/territories/states.

### Additional Notes
Modification of Gitops project to remove these environment variables will be done in a separate PR.
